### ISSUE TITLE
no output when testing body3d_multiview_detect_and_regress_img_demo.py on custom images

### ISSUE THREAD

**Issue by leonhard-yu-zhang** [open-mmlab/mmpose#1972]
Hello, when I try to test body3d_multiview_detect_and_regress_img_demo.py on custom images, there is no output of the skeleton. I've followed the tutorial (https://mmpose.readthedocs.io/en/latest/demo.html#d-multiview-human-pose-demo) to organize my data and rewrote the camera_parameters.json with the extrinsic and intrinsic parameters of my own realsense D455 cameras. There is always at least one person in my custom images. I notice that when debugging the x,y,z of pose_3d in the show_result function are all zero. Could you give some advice?

**Comment by wusize** [comment]
Hi! Did you finetune the model on your dataset?

**Comment by leonhard-yu-zhang** [comment]
No. I'm a beginner in deep learning. Will other parameters such as the size of feature volume also affect the output?

**Comment by wusize** [comment]
The method VoxelPose has poor generalization ability. It cannot generate reasonable results when inferencing on a custom dataset without any finetuning.

**Comment by Taylorminer** [comment]
> The method VoxelPose has poor generalization ability. It cannot generate reasonable results when inferencing on a custom dataset without any finetuning.

How to finetune the model on a custom dataset ?
